charts/backingservices/README.md

```diff
@@ -13,14 +13,14 @@ You can provision this SRS into your `pega` environment namespace, with the SRS
-|-----------------------|-------------|-----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| < 8.6                 | NA          | NA                    | SRS can be used with Pega Infinity 8.6 and later                                                                                                                                                                                                                                                                  |
-| \>= 8.6               | \>=1.12.0   | 7.10.2 or 7.16.3      | Pega Infinity 8.6 and later supports using a Pega-provided platform-services/search-n-reporting-service Docker Image that is tagged with version 1.12.0 or later (Pega recommends using 1.17.10 or later). All SRS Docker image versions are certified to support Elasticsearch version 7.10.2 and 7.16.3 as well |
+| Pega Infinity version | SRS version | ElasticSearch version | Description                                                                                                                                                                                                                                                                                                                                   |
+|-----------------------|-------------|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| < 8.6                 | NA          | NA                    | SRS can be used with Pega Infinity 8.6 and later                                                                                                                                                                                                                                                                                              |
+| \>= 8.6               | \>=1.12.0   | 7.10.2 or 7.16.3      | Pega Infinity 8.6 and later supports using a Pega-provided platform-services/search-n-reporting-service Docker Image that is tagged with version 1.12.0 or later (Pega recommends using the latest available SRS image - 1.17.10 or later). <br/>All SRS Docker image versions are certified against Elasticsearch versions 7.10.2 and 7.16.3 |
```

Comment:
Let's just take 1.12.0 out of the equation and use 1.17.10 as the minimum, since it includes log4j fixes and it's just very complicated to unknowning clients who need minimal guidance...

| \>= 8.6               | \>=1.17.10   | 7.10.2 or 7.16.3      | Pega Infinity 8.6 and later supports using a Pega-provided platform-services/search-n-reporting-service Docker Image. While all SRS Docker image versions starting at 1.12.0 are certified against Elasticsearch versions 7.10.2 and 7.16.3, Pega recommends using the latest available SRS image - 1.17.10 or later. |